cohort_4/playbook/ACTIVITIES.md

```diff
@@ -88,6 +89,109 @@
 
 <!-- markdown-toc end -->
 
+# Creating a safe space
+
+The one rule that we have at Hack Camp is that we make it a safe space.
+We all know how bad it feels when we are made fun of or when someone makes
+us feel uncomfortable.
+
+Below are some scenarios which may occur and how to handle them:
```

Comment:
You say that there are multiple scenarios below, but there's only one. It wasn't super clear in my previous comment, but I recommend making the below scenario part of the main "Creating a safe space" headline itself.